src/GitHub.Api/Git/Repository.cs

```diff
@@ -149,30 +160,151 @@ private void RepositoryManager_OnStatusUpdated(GitStatus status)
             CurrentStatus = status;
         }
 
-        private void RepositoryManager_OnActiveRemoteChanged(ConfigRemote? remote)
+        private void RepositoryManager_OnLocksUpdated(IEnumerable<GitLock> locks)
         {
-            CurrentRemote = remote;
+            CurrentLocks = locks;
+            OnLocksChanged?.Invoke(CurrentLocks);
```

Comment:
Currently we assume every lock update is a full change.
Should we compare and fire changed when appropriate?